portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -84,7 +84,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     if (sanitizedCommentText.trim().length() == 0 || username.trim().length() == 0) {
       response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
       return;
-    }
+    } 
```

Comment:
Is this a mistype?

Reply:
Yep, I decided not to edit the servlet in this PR. Not sure why it's still showing up as a diff though.